backend/hct_mis_api/apps/household/views.py

```diff
@@ -112,7 +112,7 @@ def get(self, request: Request) -> Response:
 
         try:
             data = get_household_or_individual(tax_id, registration_id, business_area_code)
-        except Exception as exception:
-            return Response({"status": "not found", "error_message": str(exception)}, status=404)
+        except Exception:  # pragma: no cover
+            return Response({"status": "not found", "error_message": "Household not Found"}, status=404)
```

Comment:
can we capture the exception in sentry?

Reply:
I added logging, but changing exception to something more specific is not possible, without wrewriting this functions
<img width="1077" alt="image" src="https://github.com/user-attachments/assets/dc300be0-0ba8-499d-9bac-7187f0314fd7">

Reply:
I see Imported Document here... I'm a little scared!